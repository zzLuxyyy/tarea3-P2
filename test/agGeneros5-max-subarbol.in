crearAGGeneros
insertarAGGeneros -1 44 Comedia
obtenerSubarbolAGGeneros 44
obtenerMaxAGGeneros
insertarAGGeneros 44 17 Historica
insertarAGGeneros 44 15 Drama
insertarAGGeneros 17 23 Terror
insertarAGGeneros 17 40 Comedia
obtenerSubarbolAGGeneros 17
obtenerMaxAGGeneros
insertarAGGeneros 23 3 Terror
insertarAGGeneros 15 88 Terror
obtenerSubarbolAGGeneros 88
obtenerMaxAGGeneros
liberarAGGeneros
Fin
