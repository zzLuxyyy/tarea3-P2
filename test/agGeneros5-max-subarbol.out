1>Se ejecutó 'crearTAGGeneros' exitosamente.
2>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre -1 e idGenero 44.
3>Árbol de géneros:
44 - Comedia
4>El maximo genero es: 44.
5>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 44 e idGenero 17.
6>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 44 e idGenero 15.
7>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 17 e idGenero 23.
8>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 17 e idGenero 40.
9>Árbol de géneros:
17 - Historica
    40 - Comedia
    23 - Terror
10>El maximo genero es: 44.
11>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 23 e idGenero 3.
12>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 15 e idGenero 88.
13>Árbol de géneros:
88 - Terror
14>El maximo genero es: 88.
15>Se ejecutó 'liberarTAGGeneros' exitosamente.
16>Fin.
